src/main/resources/logback.xml

```diff
@@ -55,7 +65,7 @@
   </appender>
 
   <root level="WARN">
-    <appender-ref ref="${profile}" />
+    <appender-ref ref="CLOUD" />
```

Comment:
This has the effect of hard coding the CLOUD profile for all logging - is this intentional?